client/my-sites/people/people-invite-details/index.jsx

```diff
@@ -73,7 +73,7 @@ export class PeopleInviteDetails extends React.PureComponent {
 		);
 		const clearMessage = translate(
 			'If you no longer wish to see this record, you can clear it. ' +
-				'The person will still remain a member of this site.'
+				"It will not effect the person's membership status."
```

Comment:
I could be wrong here and I'm sure that it'll be confirmed with the Copy Review anyway, but just to flag it up to ensure it's not accidentally missed, I thought that I'd highlight what might be a typo! 

```suggestion
				"It will not affect the person's membership status."
```

Reply:
Ha! I even changed that away from `affect` I never know. Thanks